{{cookiecutter.project_slug}}/.prettierignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
Could you explain why we need this one? It seems a bit unrelated (I don't see prettier configured). 

I'm asking because In the pre-commit config, we exclude Django templates from prettier in another way:

https://github.com/cookiecutter/cookiecutter-django/blob/2e561ed6c4c7ed8950334eaa63eca70a32dac055/%7B%7Bcookiecutter.project_slug%7D%7D/.pre-commit-config.yaml#L25

Maybe we should make the 2 exclusins more consistent?